test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+	private Object itemIsNull;
+	private Object itemIsNotNull = new Object();
+
+	@Test
+	public void isAnyNull_checksCorrectlyWithNoArguments() {
+		assertFalse(Utils.isAnyNull());
+	}
+
+	@Test
+	public void isAnyNull_checksAnItemCorrectly() {
+		assertTrue(Utils.isAnyNull(itemIsNull));
+		assertFalse(Utils.isAnyNull(itemIsNotNull));
+	}
+
+	@Test
+	public void isAnyNull_checksMultipleItemsCorrectly() {
+		assertTrue(Utils.isAnyNull(itemIsNull, itemIsNotNull));
+	}
+
+	@Test
+	public void elementsAreUnique_checksCorrectlyWithEmptyCollection() {
+		assertTrue(Utils.elementsAreUnique(new ArrayList<Integer>()));
+	}
+
+	@Test
+	public void elementsAreUnique_checksCorrectlyUniqueCollection() {
+		ArrayList<Integer> integers = new ArrayList<Integer>();
+		integers.add(1);
+		integers.add(2);
+		integers.add(3);
```

Comment:
You can use `List<Integer> list = Arrays.asList(1, 1, 1, 2)` to create and initialize a list in one line.